esp/platform/persistent.cpp

```diff
@@ -206,6 +208,7 @@ class CPersistentHandler : implements IPersistentHandler, implements ISocketSele
         }
         else if (m_notify != nullptr)
         {
+            bool reachingQuota = false;
```

Comment:
Should this be "reachedQuota"?

Reply:
Changed reachingQuota to reachedQuota (also changed another place "quotaReached" to "reachedQuota" for consistency) @afishbeck